src/kixi/heimdall/handler.clj

```diff
@@ -98,8 +98,8 @@
   {:status 200 :body {:type "groups" :items (service/groups (dynamodb req) (get (:params req) "id"))}})
 
 (defn get-all-groups [req]
-  (let [dex (Integer/parseInt (get-in req [:params :index] "0"))
-        cnt (min (Integer/parseInt (get-in req [:params :count] "100")) 100)
+  (let [dex (Integer/parseInt (get-in req [:params "index"] "0"))
+        cnt (min (Integer/parseInt (get-in req [:params "count"] "100")) 100)
```

Comment:
these are going to fail horribly if [:params "count"] and [:params "index"] aren't something that you can parse to an integers.

Factor out the parse function into something that will return what you want? `(str->int in-str default)`?